homeassistant/components/zwave_mqtt/binary_sensor.py

```diff
@@ -0,0 +1,400 @@
+            8,
+            9,
+            10,
+            11,
+            12,
+            13,
+            14,
+            15,
+            16,
+            17,
+            18,
+            19,
+            20,
+            21,
+        ],
+        NOTIFICATION_DEVICE_CLASS: None,
+    },
+    {
+        # Index 13: Home Health - Value Id's 1,2,3,4,5
+        NOTIFICATION_TYPE: ValueIndex.NOTIFICATION_APPLIANCE,
+        NOTIFICATION_VALUES: [1, 2, 3, 4, 5],
+        NOTIFICATION_DEVICE_CLASS: None,
+    },
+    {
+        # Index 14: Siren
+        NOTIFICATION_TYPE: ValueIndex.NOTIFICATION_SIREN,
+        NOTIFICATION_VALUES: [1],
+        NOTIFICATION_DEVICE_CLASS: DEVICE_CLASS_SOUND,
+    },
+    {
+        # Index 15: Water valve
+        # ignore non-boolean values
+        NOTIFICATION_TYPE: ValueIndex.NOTIFICATION_WATER_VALVE,
+        NOTIFICATION_VALUES: [3, 4],
+        NOTIFICATION_DEVICE_CLASS: DEVICE_CLASS_PROBLEM,
+    },
+    {
+        # Index 16: Weather
+        NOTIFICATION_TYPE: ValueIndex.NOTIFICATION_WEATHER,
+        NOTIFICATION_VALUES: [1, 2],
+        NOTIFICATION_DEVICE_CLASS: DEVICE_CLASS_PROBLEM,
+    },
+    {
+        # Index 17: Irrigation
+        # ignore non-boolean values
+        NOTIFICATION_TYPE: ValueIndex.NOTIFICATION_IRRIGATION,
+        NOTIFICATION_VALUES: [1, 2, 3, 4, 5],
+        NOTIFICATION_DEVICE_CLASS: None,
+    },
+    {
+        # Index 18: Gas
+        NOTIFICATION_TYPE: ValueIndex.NOTIFICATION_GAS,
+        NOTIFICATION_VALUES: [1, 2, 3, 4],
+        NOTIFICATION_DEVICE_CLASS: DEVICE_CLASS_GAS,
+    },
+    {
+        # Index 18: Gas
+        NOTIFICATION_TYPE: ValueIndex.NOTIFICATION_GAS,
+        NOTIFICATION_VALUES: [6],
+        NOTIFICATION_DEVICE_CLASS: DEVICE_CLASS_PROBLEM,
+    },
+]
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up Z-Wave binary_sensor from config entry."""
+
+    @callback
+    def async_add_binary_sensor(values):
+        """Add Z-Wave Binary Sensor."""
+        sensors_to_add = []
+
+        if values.primary.type == ValueType.LIST:
+            # we convert Notification values into binary sensors
+            # https://github.com/OpenZWave/open-zwave/blob/master/config/NotificationCCTypes.xml
+            for list_value in values.primary.value["List"]:
+                # check if we have a mapping for this value
+                for item in NOTIFICATION_SENSORS:
+                    if item[NOTIFICATION_TYPE] != values.primary.index:
+                        continue
+                    if list_value["Value"] not in item[NOTIFICATION_VALUES]:
+                        continue
+                    sensors_to_add.append(
+                        ZWaveListValueSensor(
+                            # required values
+                            values,
+                            list_value["Value"],
+                            # optional values
+                            item.get(NOTIFICATION_DEVICE_CLASS),
```

Comment:
Looks like this item is always present.
```suggestion
                            item[NOTIFICATION_DEVICE_CLASS],
```

Reply:
Hmmm, I marked it as optional in the comments somehow. Maybe it was not so optional after all ;-)